encrypt.go

```diff
@@ -4,6 +4,9 @@ type CompactTLSAssets struct {
 	APIServerCACrt    string
 	APIServerKey      string
 	APIServerCrt      string
+	WorkerCACrt       string
```

Comment:
I am not sure where the wording `CACrt` comes from but we wanted to have the scheme `ca`, `crt`, `key`. So it should be `WorkerCA` IMO.